### PR TITLE
Neo.VM.3.0.0-CI00190

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

File: src/neo/SmartContract/ContractParametersContext.cs [hunk 1]
```diff
@@ -144,16 +144,16 @@ public bool AddSignature(Contract contract, ECPoint pubkey, byte[] signature)
...
+                        case (byte)OpCode.PUSHINT8:
                             ++i;
                             break;
-                        case 2:
+                        case (byte)OpCode.PUSHINT16:
                             i += 2;
                             break;
                     }
-                    while (contract.Script[i++] == 33)
+                    while (contract.Script[i++] == (byte)OpCode.PUSHDATA1)
```
Comment (shargon) [inline src/neo/SmartContract/ContractParametersContext.cs]:
```suggestion
                    while (contract.Script[i++] == (byte)OpCode.PUSHDATA1) && contract.Script[i++] == 33
```

Comment (erikzhang) [reply src/neo/SmartContract/ContractParametersContext.cs]: It is checked in the next line.

Comment (shargon) [reply src/neo/SmartContract/ContractParametersContext.cs]: We take 33, but it could be more than this

Comment (erikzhang) [reply src/neo/SmartContract/ContractParametersContext.cs]: What do you mean? I have test this method.

Comment (shargon) [reply src/neo/SmartContract/ContractParametersContext.cs]: The problem is that we don't  check that the next byte is exactly 33

Comment (erikzhang) [reply src/neo/SmartContract/ContractParametersContext.cs]: It has been checked here:

https://github.com/neo-project/neo/blob/8d2b0ba79ce6833177ae898ba00c6990facfe93c/src/neo/SmartContract/ContractParametersContext.cs#L133

**Pull request merged**